applications/frontend/src/components/workspace/Workspaces.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import FolderIcon from '@material-ui/icons/Folder';
+
+import { RootState } from '../../store/rootReducer';
+import { Workspace } from '../../types/workspace';
+
+const useStyles = makeStyles((theme) => ({
+  paper: {
+    padding: theme.spacing(1),
+    margin: theme.spacing(1),
+    textAlign: 'left',
+    maxWidth: "20%"
```

Comment:
may we use a grid for this?

Reply:
I started with a grid but it didn't look as good, can't remember anymore why